cluster-provision/images/vm-image-builder/alpine-cloud-init/packages

```diff
@@ -1,10 +1,3 @@
-chrony
-less
-logrotate
-openssh
-ssmtp
-sudo
 cloud-init
-qemu-guest-agent
```

Comment:
Is it necessary for kubevirt tests?

Reply:
Not yet, let's go step by step, so we have the fastest alpline cloud-init image possible.